DigitalEcosystem/refined/perovskite_comparison/perovskites.py

```diff
@@ -135,47 +135,39 @@ def unscale(arr):
 train_pred_y = model.predict(train_x)
 test_pred_y = model.predict(test_x)
 
-plt.scatter(x=unscale(train_pred_y), y=unscale(train_y), label="Train")
-plt.scatter(x=unscale(test_pred_y), y=unscale(test_y), label="Test")
-plt.xlabel("Predicted")
-plt.ylabel("Actual")
-plt.legend()
+def create_partiy_plot(train_yhat, test_yhat):
```

Comment:
Yay, "partiy"! Pls fix.

Any chance we can reuse the content between `save_parity_plot`, `create_partiy_plot`, `create_multi_parity_plot`?

Reply:
Sure, added